config/src/parameters/actual.rs

```diff
@@ -71,16 +71,24 @@ impl Root {
 pub struct Common {
     pub chain_id: ChainId,
     pub key_pair: KeyPair,
-    pub p2p_address: SocketAddr,
+    pub p2p: P2p,
```

Comment:
It feels confusing, because while the network address is a _common_ option, the idle timeout is not. Could you re-arrange them?

Reply:
How do you think i should rearrange them?

Reply:
Replied in other thread generally.

Reply:
> How do you think i should rearrange them?

You could put `P2p` into the `Root` struct instead of `Common`